rollup.config.js

```diff
@@ -1,16 +1,20 @@
 import json from 'rollup-plugin-json';
-import babel from 'rollup-plugin-babel';
-import includePaths from 'rollup-plugin-includepaths';
-import url from 'rollup-plugin-url';
+import typescript from 'rollup-plugin-typescript2';
+import copy from 'rollup-plugin-copy-glob';
 
 export default {
-  entry: 'src/mollie.js',
-  format: 'cjs',
-  dest: 'dist/mollie.js',
+  input: 'src/mollie.ts',
+  external: ['fs', 'https', 'path', 'axios', 'qs', 'lodash/omit', 'lodash/toPlainObject'],
```

Comment:
Shouldn't this be `lodash`? We are only importing from `lodash`, never from `lodash/<function>` directly. 

Are we sure we are not importing the whole `lodash` package? Here are some suggestions https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

Reply:
Correct, thanks! I'll update it.